front/README.md

```diff
@@ -76,10 +70,14 @@ For example, to add XYZ to the `package.json` file in the `front` folder while d
 Afterwards:
 
 1. stop docker-compose and run `docker compose up` again
-2. run `pnpm` in your local front folder (so that your tooling like VSCode also picks up the changes, like new TS types etc)
+2. run `pnpm i` in your local front folder (so that your tooling like VSCode also picks up the changes, like new TS types etc)
```

Comment:
I do not find this recommendation in BretFisher recommendations. Does running this not mean to install the packages als locally outside the docker container?

Reply:
You need to, just like you need Python locally to have your tools available. If you are developing for the frontend, having Node.js et al is a given.